expo/gbe/models.py

```diff
@@ -344,6 +344,10 @@ def schedule(self):
             resources_allocated__resource__worker___item=self)]
         return sorted(set(events), key=lambda event: event.start_time)
 
+    def volunteer_schedule(self, conference=None):
+        conference = conference or Conference.current_conf()
```

Comment:
I think this is a good issue to address, but not one I want to address here. You're right, this pattern has got a bit out of control and we should figure out a way to address it.